src/main/java/org/sagebionetworks/bridge/udd/config/SpringConfig.java

```diff
@@ -67,14 +65,24 @@ public Table ddbStudyTable() {
         return ddbClient().getTable(ddbPrefix() + "Study");
     }
 
+    @Bean(name = "ddbSynapseMapTable")
+    public Table ddbSynapseMapTable() {
+        return ddbClient().getTable(environmentConfig().get("synapse.map.table"));
+    }
+
     @Bean(name = "ddbUploadTable")
     public Table ddbUploadTable() {
         return ddbClient().getTable(ddbPrefix() + "Upload2");
     }
 
-    @Bean(name = "ddbUploadTableIndex")
-    public Index ddbUploadTableIndex() {
-        return ddbUploadTable().getIndex("healthCode-uploadDate-index");
+    @Bean(name = "ddbUploadSchemaTable")
+    public Table ddbUploadSchemaTable() {
+        return ddbClient().getTable(ddbPrefix() + "UploadSchema");
```

Comment:
Good idea. Waiting on when that pull request makes it into the maven bucket. Tracking using https://sagebionetworks.jira.com/browse/BRIDGE-825